### DISCUSSION TITLE
Rounded tab bar buttons (Firefox 137 and newer)

### DISCUSSION THREAD

**Discussion by QNetITQ** [QNetITQ/WaveFox#180] [Announcements]
**There is a side effect! Some icons may be cut off at the edges**

```
/* ---------- Rounded tab bar buttons ---------- */

.browser-titlebar
{
    --toolbarbutton-border-radius: 50px !important;
    --tab-border-radius: 50px !important;
}

.tabbrowser-tab:not([hidden]) + #tabbrowser-arrowscrollbox-periphery > #tabs-newtab-button,
.tabbrowser-tab:not([hidden]) + .tabbrowser-tab[hidden] + #tabbrowser-arrowscrollbox-periphery > #tabs-newtab-button
{
    margin-inline-start: calc(-1 * var(--tab-first-last-inline-margin, 0px)) !important;
}
```

**Comment by tockudex** [comment]
Thanks for the info, it's exactly what I was looking for !

I think it'd be great to integrate these line (and the others from the discussion) into your project, as they refresh Firefox's style and allow for an easier turn on/off mechanism.

**Comment by jukfiuune** [comment]
Can you make something similar, but the new tab button is like in australis (the button matches the tab style)?

**Reply by QNetITQ** [reply]
There have always been certain difficulties with this. I'll try a little later.
